File: ConsultasVisuales/src/app/model/county-natality-search-response.ts
export interface CountyNatalitySearchResponse {

  Codigo_FIPS_Del_Condado: string;
  Nacimientos: string;
  Edad_Promedio_de_la_Madre: string;
  Edad_OE_Promedio_Gestacional_Semanas: string;
  Edad_LMP_Promedio_Gestacional_Semanas: string;
  Peso_Promedio_de_Nacimiento_gramos: string;
  IMC_Promedio_Previo_al_Embarazo: string;
  Número_Promedio_de_Semanas_Prenatales: string;
  [key: string]: string;
}

File: consultas-bigQuery-gcp/src/main/java/com/medici/app/dto/CountyNatalitySearchResponse.java
package com.medici.app.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CountyNatalitySearchResponse {

   // private String Codigo_FIPS_Del_Condado;
    private String Nacimientos;
    private String Edad_Promedio_de_la_Madre;
    private String Edad_OE_Promedio_Gestacional_Semanas;
    private String Edad_LMP_Promedio_Gestacional_Semanas;
    private String Peso_Promedio_de_Nacimiento_gramos;
    private String IMC_Promedio_Previo_al_Embarazo;
    private String Número_Promedio_de_Semanas_Prenatales;




}
